src/components/manage/Form/InlineForm.jsx

```diff
@@ -92,11 +129,7 @@ const InlineForm = ({
               id={field}
               fieldSet={defaultFieldset.title.toLowerCase()}
               focus={index === focusIndex}
-              value={
-                'default' in schema.properties[field]
-                  ? formData[field] ?? schema.properties[field].default
-                  : formData[field]
-              }
+              value={formData[field]}
```

Comment:
I think (but this needs to be checked) that this should be `formData[field] || null`, otherwise we're creating input components initially with 'undefined', then switching them to controlled inputs.